🐙 source-recurly: run up-to-date pipeline [2024-12-11]

airbyteio opened the pull request:
# Update source-recurly

This PR was autogenerated by running `airbyte-ci connectors --name=source-recurly up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- PATCH bump source-recurly version to 1.3.1: Successful

- Build source-recurly docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=49091) <br/>All committers have signed the CLA.

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
PATCH bump source-recurly version to 1.3.1
Build source-recurly docker image for platform(s) linux/amd64, linux/arm64
Create or update pull request on Airbyte repository
Add changelog entry
